plugins/com.google.cloud.tools.eclipse.appengine.validation/plugin.properties

```diff
@@ -5,7 +5,6 @@ extension.name=App Engine Source Code Validation
 appengine.web=App Engine Web XML Validator
 pom.validator=Pom Validator
 appengine.blacklist.marker=App Engine Configuration Problem
```

Comment:
We should rename `appengine.blacklist.marker`?

Reply:
Yes, we should. I'm splitting the big PR in #3344 that does this into smaller pieces to try to narrow down why some parts of that aren't working. This is just the first piece.

Reply:
👍